Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnSelect.hs

```diff
@@ -91,12 +96,26 @@ onSelect DOnSelectReq {..} = do
     QQuote.createMany quotes
     QPFS.updateStatus searchRequest.riderId DPFS.DRIVER_OFFERED_QUOTE {estimateId = estimate.id, validTill = searchRequest.validTill}
     QEstimate.updateStatus estimateId $ Just DEstimate.GOT_DRIVER_QUOTE
-  Notify.notifyOnDriverOfferIncoming estimateId quotes person
+
+  if estimate.autoAssignEnabledV2
+    then do
+      let autoAssignQuote = head quotes
+      dConfirmRes <- DConfirm.confirm personId autoAssignQuote.id
+      becknInitReq <- ACL.buildInitReq dConfirmRes
+      handle (errHandler dConfirmRes.booking) $
+        void $ withShortRetry $ CallBPP.init dConfirmRes.providerUrl becknInitReq
+      Notify.notifyOnDriverOfferIncoming estimateId quotes person
```

Comment:
`Notify.notifyOnDriverOfferIncoming` function can be moved outside of `if` clause

Reply:
In case of autoAssign we should not send driver offer incoming notification.